.clang-format

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Should we be committing this if it's still tentative and too invasive? Is it usable, or is it confusing if you have to go in and fix the results after you've applied it? Maybe it should be saved for a future PR

Reply:
It's too invasive, it reformats too much stuff. I just wanted to quickly convert spaces to tabs. If we could convince the config to do just that and nothing else it would be good to merge.